.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,12 @@
 
 ----
 
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our [Contributing guide?](../CONTRIBUTING.md)
+
+### Adding new Dependencies:
```

Comment:
```suggestion
### Adding new Unconventional Dependencies:
```

Reply:
maybe it's okay for this to say 'new dependencies' since we want more people to check out what we're talking about here. will leave it up to you.